bin/coadd_zint

```diff
@@ -64,9 +69,12 @@ for f in args.data:
             we[p] = we_aux[i]
     h.close()
 
-    h = fitsio.FITS(f.replace('cf','dmat'))
-    dm += h[1]['DM'][:]*wet_aux[:,None]
-
+    if not args.no_dmat:
+        h = fitsio.FITS(f.replace('cf','dmat'))
+        dm += h[1]['DM'][:]*wet_aux[:,None]
+        h.close()
+    else:
+        dm += sp.identity(nb.shape[0])*wet_aux[:,None]
```

Comment:
remove these two lines.

Reply:
Sorry exactly which two lines are you referring to?

Reply:
remove lines 76 and 77